.circleci/config.yml

```diff
@@ -109,10 +113,67 @@ jobs:
           path: ~/project
       - attach_workspace:
           at: /tmp/workspace
-      - run: ../scripts/pack_deb.sh -v 0.1 -f /tmp/workspace/administration/build -d "Administration backend for the Ehrenamtskarte app" -n "eak-administration"
+      - run: ~/project/scripts/pack_deb.sh -v "0.$CIRCLE_BUILD_NUM" -f /tmp/workspace/administration/build -d "Administration backend for the Ehrenamtskarte app" -n "eak-administration"
       - run: |
           mkdir -p /tmp/artifacts/debs
-          cp *.deb /tmp/artifacts/debs
+          mv *.deb /tmp/artifacts/debs
+      - store_artifacts:
+          path: /tmp/artifacts
+      - persist_to_workspace:
+          root: /tmp/artifacts
+          paths:
+            - debs/*.deb
+
+  styles-pack:
+    docker:
+      - image: cimg/base:2020.01
+    working_directory: ~/project/mapbox/styles
+    steps:
+      - checkout:
+          path: ~/project
+      - run:
+          name: "Download glyphs"
+          command: |
+            wget -O /tmp/glyphs.zip https://github.com/orangemug/font-glyphs/archive/gh-pages.zip
+            unzip /tmp/glyphs.zip
+            mv font-glyphs* font-glyphs
+            rm -f /tmp/glyphs.zip
```

Comment:
What are the glyphs for?

Reply:
They are the fonts used on the map

Reply:
Ah, but they have nothing to do with martin, right?

Reply:
Nope, they are just there because of this branch containing the changes of #237 already.